pkgs/applications/kde/kigo.nix

```
@@ -3,8 +3,8 @@
 mkDerivation {
   name = "kigo";
   meta = with lib; {
-    homepage = "https://kde.org/applications/en/games/org/kde.kigo";
-    description = "Kigo est une implémentation libre du jeu de Go";
+    homepage = "https://kde.org/applications/en/games/org.kde.kigo";
+    description = "Kigo is an open-source implementation of the popular Go game";
```

```suggestion
    description = "An open-source implementation of the popular Go game";
```

Following CONTRIBUTING.md, *`meta.description` should: Not start with the package name.*